requirements.txt

```diff
@@ -0,0 +1,17 @@
+git+https://github.com/arcalot/arcaflow-plugin-sdk-python@d6a4d98017762fa488d308fbf344372fcb4f3629
```

Comment:
Now that the SDK is released, do we need this?

Reply:
we do not, but we're in the process of updating the sdk release version right now, and I am going to make another pull request to update this dependency

Reply:
No, I am about to change it :)